python/ray/dashboard/subprocesses/handle.py

```diff
@@ -117,83 +112,74 @@ def __str__(self):
             self.incarnation, self.process.pid if self.process else None
         )
 
-    def start_module(self, start_dispatch_parent_bound_messages_thread: bool = True):
-        """
-        Params:
-        - start_dispatch_parent_bound_messages_thread: used for testing.
-        """
-        self.next_request_id = 0
-        self.child_bound_queue = multiprocessing.Queue()
-        self.parent_bound_queue = multiprocessing.Queue()
-        self.active_requests.pop_all()
-        self.process = multiprocessing.Process(
+    def __del__(self):
+        logger.info("SubprocessModuleHandle is being deleted")
+        if self.loop.is_running():
+            self.loop.run_until_complete(self.destroy_module())
```

Comment:
Do we need this? This only happens when we stop the cluster?

Reply:
Let me examine this again. I copied this destructor from Ruiyang's PR.